doc/user_guide/examples.rst

```diff
@@ -397,14 +398,33 @@ code that is output is the same as the original (but looks different
 as it has been translated to PSyIR and then output by the PSyIR
 fortran back-end).
 
-Example 16: Kernel Extraction
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The subdirectory ``full_example_extract`` contains a runnable example
-of LFRic code that creates a NetCDF file with the input and output
-parameters of a simple kernel. This example requires the installation
-of a Fortran compiler and NetCDF development environment. After compilation
-of the code, you can run the example. which will produce a NetCDF file:
+Example 17: Runnable Simplified LFRic Examples
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This directory contains three simplified LFRic examples, that can be
+compiled and executed - a suitable Fortran compiler is of course
+required. The examples are using a subset of the LFRic infrastructure
+library, which is contained in PSyclone and which has been slightly
+modified to make it easier to create stand-alone, non-MPI LFRic codes.
+
+Example 17.1: Very Simple Runnable Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The subdirectory ``full_example`` contains a very simple example code
+that uses PSyclone to create two single kernels. It uses unit-testing
```

Comment:
Bad wording, I think I meant two invokes (one of which uses only builtins). I am renaming this. 

I've renamed the file in the three new examples.

I agree that coding style should be specified if we want to follow LFRIc rules. Considering that we have over 80 files in testing called ``testkern_*``, and we have [Fortran coding style rules](https://github.com/stfc/PSyclone/blob/master/doc/user_guide/fortran_naming_conventions.rst) already (which does not specify the kernel naming), I don't want to change this as a 'side' effect of an essentially unrelated PR. I've opened #1159 to discuss what to do with the testing files if we follow the LFRic coding style.

Reply:
Since I realised at the telco that I misunderstood (you only wanted to have these examples to follow the coding style, not all LFRic tests etc), I have closed #1159. Renaming in these three examples was done.